include/spore/codegen/codegen_app.hpp

```diff
@@ -89,8 +89,45 @@ namespace spore::codegen
         {
             std::vector<std::filesystem::path> inputs = glob::rglob(step.input);
 
+            // search for templates starting at the working directory and then going
+            // through configured template prefix paths.
+            SPDLOG_DEBUG("searching for templates");
+            std::vector<std::string> templates;
+
+            for (const auto& template_ : step.templates)
+            {
+                bool template_found = false;
+
+                SPDLOG_DEBUG("  search {}", template_);
```

Comment:
I usually format logs this way:
```cpp
SPDLOG_DEBUG("message, key1=value1 key2=value2");
```

Reply:
Are you ok with the indentation? This makes it clear from the log it's relative to the current loop iteration.
Let me know if you want me to remove that too.